Rakefile.rb

```diff
@@ -1,30 +1,49 @@
 require 'json'
 require 'ohai'
 
-PROJECT_DIR='/dd-agent-omnibus'
-
 @ohai = Ohai::System.new.tap { |o| o.all_plugins(%w{platform}) }.data
 
+def linux?()
+  %w(rhel debian fedora suse gentoo slackware arch exherbo).include? @ohai['platform_family']
+end
+
+def osx?()
+  @ohai['platform_family'] == 'mac_os_x'
+end
+
+def windows?()
+  @ohai['platform_family'] == 'windows'
+end
+
+
+if windows?
+  PROJECT_DIR='c:\dd-agent-omnibus'
+  FSROOT="/c/"
+else
+  PROJECT_DIR='/dd-agent-omnibus'
+  FSROOT="/"
+end
+
 namespace :agent do
   desc 'Cleanup generated files'
   task :clean do |t|
     puts "Clean up generated files"
-    sh "rm -rf /var/cache/omnibus/pkg/*"
-    sh "rm -f /etc/init.d/datadog-agent"
-    sh "rm -rf /etc/dd-agent"
-    sh "rm -rf /opt/datadog-agent"
+    sh "rm -rf #{FSROOT}var/cache/omnibus/pkg/*"
+    sh "rm -f #{FSROOT}etc/init.d/datadog-agent"
+    sh "rm -rf #{FSROOT}etc/dd-agent"
+    sh "rm -rf #{FSROOT}opt/datadog-agent"
     t.reenable
   end
 
   desc 'Pull the integrations repo'
   task :'pull-integrations' do
     integration_branch = ENV['INTEGRATION_BRANCH'] || 'master'
 
-    sh "rm -rf /#{ENV['INTEGRATIONS_REPO']}"
+    sh "rm -rf #{FSROOT}#{ENV['INTEGRATIONS_REPO']}"
```

Comment:
This was the existing code form @gmmeyer .  Note the only change is the FSROOT so it will execute correctly.